cmd/rhoas/main.go

```diff
@@ -128,3 +137,24 @@ func initConfig(f *factory.Factory) {
 		os.Exit(1)
 	}
 }
+
+// check if the config file is located in the old default location
+// if so, move it to the new location
+func moveConfigFile(cfg config.IConfig) error {
+	cfgPath, err := cfg.Location()
+	if err != nil {
+		return err
+	}
+	userCfgPath, err := os.UserConfigDir()
+	if err != nil {
+		return err
+	}
+	oldFilePath := filepath.Join(userCfgPath, ".rhoascli.json")
+	if os.Getenv("RHOASCLI_CONFIG") == oldFilePath {
+		return nil
+	}
+	if _, err = os.Stat(oldFilePath); err == nil {
+		return os.Rename(oldFilePath, cfgPath)
+	}
+	return nil
+}
```

Comment:
Migrating the old config file. I want to start a habit of non-breaking changes now that we have users. This function can be removed after a few releases.